src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -363,9 +369,7 @@ parseBlock (Elem e) = do
          wrapWithHeader n mBlocks = do
                       isBook <- gets jatsBook
                       let n' = if isBook || n == 0 then n + 1 else n
-                      headerText <- case filterChild (named "title") e `mplus`
-                                          (filterChild (named "info") e >>=
```

Comment:
There is no `info` element in JATS. This is legacy from the DocBook reader. As such these lines are redundant.